styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/FlyteInputsUtilsTest.java

```diff
@@ -31,10 +32,13 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.protobuf.Timestamp;
 import com.google.protobuf.util.Timestamps;
+import com.spotify.styx.model.FlyteExecConf;
+import com.spotify.styx.model.FlyteIdentifier;
 import flyteidl.core.Interface;
 import flyteidl.core.Literals;
 import flyteidl.core.Types;
 
+import java.util.Map;
 import org.junit.Test;
 
 public class FlyteInputsUtilsTest {
```

Comment:
There should be a test to check that variables in styx variables overwrite the ones in userDefinedInputs